app/models/user.rb

```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
   has_many :items, dependent: :destroy
   has_many :list_items, through: :items, dependent: :destroy
-  has_many :lists, through: :list_items, dependent: :destroy
+  has_many :lists, dependent: :destroy
```

Comment:
I like this change. Even though you **can** get all of the users lists through list_items, this is a probably an operation you will have to do a lot so it will simplify your queries if lists have a user_id foreign key and belong to a user.

Reply:
Agreed! It made more since for a user to  have the ability to create an empty list instead of having to have a list_item to associate it.